src/storage.hpp

```diff
@@ -58,13 +64,14 @@ class Storage {
 
   void insert_payload(Payload payload);
   void insert_update_session(json session_data);
-  void select_all_event_rows(list<EventRow>* event_list);
-  void select_event_row_range(list<EventRow>* event_list, int range);
-  void select_all_session_rows(list<json>* session_list);
+  void select_all_event_rows(list<EventRow> *event_list);
+  void select_event_row_range(list<EventRow> *event_list, int range);
+  void select_all_session_rows(list<json> *session_list);
```

Comment:
I think the position should be the same for both `&` and `*`.
I noticed that `&` was put near the type and the `*` near the var name.